src/signals/incident/components/form/FileInput/style.scss

```diff
@@ -1,8 +1,98 @@
+@import "config.scss";
+
+$boxSize: 100px;
+$borderWidth: 1px;
+$iconSize: 43px;
+$iconLoadingSize: 40px;
+
+
 .file-input {
+  display: flex;
+  height: calc(#{$boxSize} + (2 * #{$borderWidth}));
+
   &__preview {
+    width: calc(#{$boxSize} + (2 * #{$borderWidth}));
+    margin-right: 10px;
+
+    &--loading {
+      border: 1px solid $ams-middengrijs;
+
+      .progress-indicator {
+        font-size: 4px !important;
+        top: calc((#{$boxSize} - #{$iconLoadingSize}) / 2);
+        left: calc((#{$boxSize} - #{$iconLoadingSize}) / 2);
+        }
+    }
+
+    &-button-delete {
+      position: absolute;
+      width: 40px;
+      height: 40px;
+      bottom: 0;
+      right: 0;
+      border-style: none;
+      background-image: url("../../../../../shared/images/icon-trash.svg");
+      background-size: 27px;
+      background-repeat: no-repeat;
+      background-position: 6px 6px;
+      background-color: rgba(0,0,0,0.5) !important;
+    }
+
     &-image {
-      border: 1px solid black;
-      height: 80px;
+      position: relative;
+      background-size: cover;
+      height: 100%;
+      width: 100%;
+    }
+  }
+
+  &__button {
+    border: 1px dashed $ams-middengrijs;
+    width: calc(#{$boxSize} + (2 * #{$borderWidth}));
+    margin-right: 10px;
+
+    input[type=file] {
```

Comment:
Can an `<input>` element be a child of a button element? Seems a bit odd.

Reply:
the file-input__button class is not a semantic button but a div that functions as a button.